Can't build with Puzzle Game disabled on GNU/Linux

Hello,

I am trying to build adlmidi on GNU/Linux (Debian 13 Stable) without the puzzle game.
In order to do so, I commented out https://github.com/bisqwit/adlmidi/blob/9881a230a74b9203fd597cd2029b7f45a56e1f1c/src/midiplay.cc#L23
...but that causes the build to fail with:
```
++ -I/usr/local/include -I/usr/local/include/SDL2 -D_GNU_SOURCE=1 -D_REENTRANT  -std=c++17 -pedantic -Wall -Wextra src/midiplay.cc  -Ofast -g -fopenmp -march=native  -c -o obj/midiplay.o
src/midiplay.cc: In function ‘void ParseReverb(std::string_view)’:
src/midiplay.cc:2536:18: error: ‘uint_fast64_t’ is not a member of ‘std’
 2536 |             std::uint_fast64_t a=0xF4939CE1DDA3, b=0x67EE6A00, c=0x9770000, d=0xE519FDFD45F8, result=a + d*key.size();
      |                  ^~~~~~~~~~~~~
src/midiplay.cc:303:1: note: ‘std::uint_fast64_t’ is defined in header ‘<cstdint>’; this is probably fixable by adding ‘#include <cstdint>’
  302 | #include "9x15.inc"
  +++ |+#include <cstdint>
  303 | #endif
src/midiplay.cc:2540:17: error: ‘result’ was not declared in this scope
 2540 |                 result += ch;
      |                 ^~~~~~
src/midiplay.cc:2541:78: error: ‘b’ was not declared in this scope
 2541 |                 result = (result ^ (result >> (1 + 1 * ((shift>>0) & 7)))) * b;
      |                                                                              ^
src/midiplay.cc:2542:78: error: ‘c’ was not declared in this scope; did you mean ‘ch’?
 2542 |                 result = (result ^ (result >> (1 + 1 * ((shift>>3) & 7)))) * c;
      |                                                                              ^
      |                                                                              ch
src/midiplay.cc:2544:30: error: ‘result’ was not declared in this scope
 2544 |             unsigned index = result % mod;
      |                              ^~~~~~
make: *** [make.rules:112: obj/midiplay.o] Error 1
```

--- a/src/midiplay.cc
+++ b/src/midiplay.cc
@@ -8,6 +8,7 @@
 #include <set>
 #include <cstdlib>
 #include <cstring>
+#include <cstdint>
 #include <cmath>
 #include <unistd.h>
 #include <stdarg.h>

